circleci: failed tests (13412): TestAuthentication, TestBadRequest

The following test appears to have failed:

[#13412](https://circleci.com/gh/cockroachdb/cockroach/13412):

```
W160225 19:11:29.901780 kv/dist_sender.go:609  failed to invoke ResolveIntent [/Meta2/Table/13,/Min), ResolveIntent [/Meta2/Max,/Min), ResolveIntent [/Local/Range/""/RangeTreeNode,/Min), ResolveIntent [/Local/Range/"\x93"/RangeTreeNode,/Min): failed to send RPC: too many errors encountered (1 of 1 total): server/node.go:594: node 1 stopped
I160225 19:11:29.901880 kv/range_cache.go:228  evict cached descriptor: key=/Min desc=range_id:1 start_key:"" end_key:"\223" replicas:<node_id:1 store_id:1 replica_id:1 > next_replica_id:2 
I160225 19:11:29.901923 client/db.go:488  failed batch: failed to send RPC: too many errors encountered (1 of 1 total): server/node.go:594: node 1 stopped
W160225 19:11:29.901969 storage/replica.go:1731  [node=1,store=1,range=3] failed to resolve intents: failed to send RPC: too many errors encountered (1 of 1 total): server/node.go:594: node 1 stopped
I160225 19:11:29.902167 storage/engine/rocksdb.go:137  closing in-memory rocksdb instance
--- FAIL: TestAuthentication (0.11s)
	db_test.go:254: client/rpc_sender.go:83: failed to send RPC request Server.Batch: client is unhealthy
=== RUN   TestRangeLookupWithOpenTransaction
I160225 19:11:29.917416 server/context.go:165  can't read available memory from cgroups (open /sys/fs/cgroup/memory/memory.limit_in_bytes: no such file or directory)
	setting default rocksdb cache size to 30 GiB (half of system memory)
I160225 19:11:29.921392 server/node.go:350  store store=0:0 ([]=) not bootstrapped
I160225 19:11:29.921776 rpc/clock_offset.go:154  monitoring cluster offset
I160225 19:11:29.923081 raft/raft.go:459  [group 1] 1 became follower at term 5
I160225 19:11:29.923140 raft/raft.go:235  [group 1] newRaft 1 [peers: [1], term: 5, commit: 10, applied: 10, lastindex: 10, lastterm: 5]
I160225 19:11:29.923168 raft/raft.go:539  [group 1] 1 is starting a new election at term 5
I160225 19:11:29.923195 raft/raft.go:472  [group 1] 1 became candidate at term 6
--
W160225 19:11:37.121640 kv/dist_sender.go:609  failed to invoke BeginTransaction ["a",/Min), Put ["a",/Min), Put ["b",/Min): txn "kv/dist_sender_server_test.go:585 TestPropagateTxnOnPushError" id=0b3ca512 key="a" rw=true pri=0.03315378 iso=SERIALIZABLE stat=PENDING epo=1 ts=1456427497.119399888,0 orig=1456427497.119399888,0 max=1456427497.119399888,0 failed to push "kv/dist_sender_server_test.go:559 TestPropagateTxnOnPushError.func1" id=49317553 key="b" rw=false pri=0.00724634 iso=SERIALIZABLE stat=PENDING epo=0 ts=1456427497.114971168,0 orig=1456427497.114971168,0 max=1456427497.114971168,0
I160225 19:11:37.121924 client/db.go:488  failed batch: txn "kv/dist_sender_server_test.go:585 TestPropagateTxnOnPushError" id=0b3ca512 key="a" rw=true pri=0.03315378 iso=SERIALIZABLE stat=PENDING epo=2 ts=1456427497.119399888,0 orig=1456427497.119399888,0 max=1456427497.169399888,0 failed to push "kv/dist_sender_server_test.go:559 TestPropagateTxnOnPushError.func1" id=49317553 key="b" rw=false pri=0.00724634 iso=SERIALIZABLE stat=PENDING epo=0 ts=1456427497.114971168,0 orig=1456427497.114971168,0 max=1456427497.114971168,0
I160225 19:11:37.174639 storage/replica_command.go:457  auto-gc'ed 49317553 (1 intents)
I160225 19:11:37.177195 storage/engine/rocksdb.go:137  closing in-memory rocksdb instance
--- PASS: TestPropagateTxnOnPushError (0.17s)
FAIL
FAIL	github.com/cockroachdb/cockroach/kv	27.068s
=== RUN   TestCombinable
--- PASS: TestCombinable (0.00s)
=== RUN   TestBatchSplit
--- PASS: TestBatchSplit (0.00s)
=== RUN   TestFlagsToStr
--- PASS: TestFlagsToStr (0.00s)
=== RUN   TestBatchRequestGetArg
--- PASS: TestBatchRequestGetArg (0.00s)
=== RUN   TestKeyNext
--- PASS: TestKeyNext (0.00s)

```
Please assign, take a look and update the issue accordingly.

--- a/kv/db_test.go
+++ b/kv/db_test.go
@@ -239,7 +239,6 @@ func TestKVDBTransaction(t *testing.T) {
 // TestAuthentication tests authentication for the KV endpoint.
 func TestAuthentication(t *testing.T) {
 	defer leaktest.AfterTest(t)()
-	t.Skip("#4653")
 	s := server.StartTestServer(t)
 	defer s.Stop()
 

--- a/kv/dist_sender_server_test.go
+++ b/kv/dist_sender_server_test.go
@@ -399,7 +399,6 @@ func TestReverseScanWithSplitAndMerge(t *testing.T) {
 
 func TestBadRequest(t *testing.T) {
 	defer leaktest.AfterTest(t)()
-	t.Skip("#4653")
 	s := server.StartTestServer(t)
 	db := createTestClient(t, s.Stopper(), s.ServingAddr())
 	defer s.Stop()

